htmd/queues/localqueue.py

```diff
@@ -117,7 +117,7 @@ def _createJobScript(self, fname, workdir, runsh, gpudevice=None):
                 datadir = os.path.abspath(self.datadir)
                 try:
                     os.mkdir(datadir)
-                except OSError:
+                except FileExistsError:
                     pass
```

Comment:
Or just use 
```python
os.makedirs(datadir, exist_ok=True)
```
without any exception catching...

Reply:
Ah yes. I sometimes forget that one exists... Thanks